public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>RogueLife</title>
+</head>
+<body>
+  <script src="../dist/bundle.js"></script>
```

Comment:
The path to the script `src` had an extra `/` in it, so the `Display` failed to render. I removed the slash.